react-toggle-switch/my-app/src/App.tsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+import './App.css';
+import { SwitchContainer } from './components/SwitchContainer';
+
+function App() {
+  const [isActive, setActiveStatus] = useState(false);
+
+  function handleToggle() {
+    isActive ? setActiveStatus(false) : setActiveStatus(true);
```

Comment:
You can also do `setActiveStatus(!isActive)`, so you will always set it to the opposite of the current boolean